src/components/feedbacks/FeedbackItem.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <base-box>
+    <article>
+      <p>{{}}</p>
+      <!-- status - activated in the Roadmap, hidden in the FeedbackList -->
+      <div class="feedback__votes">
+        <svg width="10" height="7" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You don't have to do it now, but later you should change the way of importing svg. It still works the same: https://v2.vuejs.org/v2/cookbook/editable-svg-icons.html you don't have to have any props there. You can just put the whole svg in component and wrap it with the `<template>` tag and then import it anywhere you wish.

Reply:
done